classes/music_album.rb

```diff
@@ -0,0 +1,18 @@
+require '../item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :name, :publish_date
+  attr_reader :items
+
+  def initialize(on_spotify, name, publish_date)
+    super(publish_date)
+    @id = id
+    @on_spotify = on_spotify
+    @name = name
+    @items = []
+  end
+
+  def can_be_archived?
+    super && @on_spotify
+  end
```

Comment:
Please make sure this is a private method in accordance to the project requirements